gulpfile.js

```diff
@@ -0,0 +1,46 @@
+// Dependencies
+var gulp       = require('gulp'),
+    handlebars = require('gulp-handlebars'),
+    wrap       = require('gulp-wrap'),
+    declare    = require('gulp-declare'),
+    concat     = require('gulp-concat'),
+    uglify     = require('gulp-uglify'),
+    minify     = require('gulp-minify-css'),
+    batch      = require('gulp-batch'),
+    watch      = require('gulp-watch');
+
+// Task that compiles templates
+gulp.task('build', function(){
+  gulp.src('templates/*.hbs')
+  .pipe(handlebars())
+  .pipe(wrap('Handlebars.template(<%= contents %>)'))
+  .pipe(declare({
+    namespace: 'CodeForDenver.templates',
+    noRedeclare: true, // Avoid duplicate declarations
+  }))
+  .pipe(concat('templates.js'))
+  .pipe(gulp.dest('build/js/'));
+});
+
+// Task to include and bundle js
+gulp.task('scripts', function(){
+  return gulp.src(['js/*.js', 'build/js/templates.js'])
```

Comment:
I figured that for now we would leave out vendor libraries from the `lib` folder since their directory structures can be different.  Let me know if you guys think I should add anything else here, though.